.github/workflows/causal_pipeline.yml

```
@@ -103,7 +103,10 @@ jobs:
           source .github/workflows/pipeline_helpers.sh
           
           run_step main_causal.estimate        estimate.yaml
-          python -m tests.pipeline.test_estimate ./outputs/causal/estimate/simple example_data/synthea_meds_causal/tuning
+          python -m tests.pipeline.test_estimate \
+          ./outputs/causal/estimate/simple \
+          example_data/synthea_meds_causal/tuning \
+          --ci_expansion_factor 2.0 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)